substrate/client/rpc-servers/src/lib.rs

```diff
@@ -103,33 +114,81 @@ pub async fn start_server<M: Send + Sync + 'static>(
 		.max_response_body_size(max_payload_out_mb.saturating_mul(MEGABYTE))
 		.max_connections(max_connections)
 		.max_subscriptions_per_connection(max_subs_per_conn)
-		.ping_interval(Duration::from_secs(30))
-		.set_middleware(middleware)
+		.enable_ws_ping(
+			PingConfig::new()
+				.ping_interval(Duration::from_secs(30))
+				.inactive_limit(Duration::from_secs(50)),
```

Comment:
The behavior before this release was: the server sends ping frames every 30 seconds and doesn't care about pong frames.
Now the behavior is: the server sends ping frames every 30 seconds and expects at least one pong frame (or event message -- correctly me if i'm wrong here) in at most 50 seconds.

Should we bump this to 60 seconds just to be sure? Not entirely convinced myself, since most RPC apis provide a response almost immediately